core/jvm/src/main/scala/zio/sql/expr.scala

```diff
@@ -158,6 +158,16 @@ trait ExprModule extends NewtypesModule with FeaturesModule with OpsModule {
       def typeTag: TypeTag[Z] = implicitly[TypeTag[Z]]
     }
 
+    sealed case class ParenlessFunctionCall0[Z: TypeTag](function: ParenlessDef[Z])
```

Comment:
I think we just need a `FunctionName` for this, then you can use it directly without calling apply